NEMO/models.py

```diff
@@ -227,6 +227,8 @@ class Tool(models.Model):
 	name = models.CharField(max_length=100, unique=True)
 	parent_tool = models.ForeignKey('Tool', related_name="tool_children_set", null=True, blank=True, help_text='Select a parent tool to allow alternate usage', on_delete=models.CASCADE)
 	visible = models.BooleanField(default=True, help_text="Specifies whether this tool is visible to users.")
+	_description = models.TextField(null=True, blank=True, default='', help_text="HTML syntax could be used")
+	_image = models.ImageField(upload_to="tool_image", blank=True, help_text="An image that represent the tool. Maximum width is 500px")
```

Comment:
```suggestion
	_image = models.ImageField(db_column="image", upload_to="tool_image", blank=True, help_text="An image that represent the tool. Maximum width is 500px")
```

Reply:
I would also suggest here changing `upload_to` to a function that would return `tool_images/{tool_name}.{ext}`
you can find an example of this in `models.py`, class `TaskImages`. the function that returns the `upload_to` filename is in `utilities.py` `get_task_image_filename`

Reply:
also if you want to restrict the size of the image, there is a function to do that in `utilities.py` named `resize_image`. you would have to do that in the clean method of `ToolAdminForm`